packages/block-library/src/image/image.js

```diff
@@ -508,7 +504,12 @@ export default function Image( {
 					: __( 'Connected to dynamic data' ),
 			};
 		},
-		[ clientId, isSingleSelected, metadata?.bindings ]
+		[
+			arePatternOverridesEnabled,
```

Comment:
```suggestion
			patternOverridesEnabled,
```
We can remove the verb from the variable in follow-up PRs.